setup.py

```diff
@@ -1,46 +1,53 @@
 #!/usr/bin/python
-
 from distutils.core import setup
-
-# from python3-distutils-extra
-from DistUtilsExtra.command import build_extra
 from DistUtilsExtra.auto import clean_build_tree
+from DistUtilsExtra.command import build_extra
 
 from landscape import UPSTREAM_VERSION
 
+# from python3-distutils-extra
```

Comment:
comment was meant to be above the DistUtilsExtra imports, but can just be removed.